docs/books/modules/user-guide/viewing-log-entries.adoc

```diff
@@ -50,6 +50,8 @@ Wed Jun  7 17:49:32 2019 SERVER (none) [2]:0 -> @flow(19) [next-incoming-id=1, i
 ----
 --
 
+NOTE: `vhost` entries are only logged if `multiTenant` is set to `true` in the router configuration file.
+
```

Comment:
FYI, I don't know if it's necessary in this case, but if you want to refer to the default path and name of the config file, you can use this attribute:

https://github.com/apache/qpid-dispatch/blob/df22f36b9d45ccf9c555f847919c911e6c6d57e8/docs/books/common/attributes.adoc#L46